lib/que/migrations/6/up.sql

```diff
@@ -2,6 +2,7 @@ ALTER TABLE que_jobs
   ADD COLUMN kwargs JSONB NOT NULL DEFAULT '{}';
 
 CREATE INDEX que_jobs_kwargs_gin_idx ON que_jobs USING gin (kwargs jsonb_path_ops);
+DROP INDEX que_poll_idx;
```

Comment:
Worth renaming the new index to this as well?

Reply:
Ah, nice, I didn't know you could do that